src/index.ts

```diff
@@ -63,5 +66,8 @@ export {
   FileUploadButtonProps,
   SpinnerProps,
   TabItem,
-  TabProps
+  TabProps,
+  ToastProps,
+  useToaster,
+  useDisplayToast
```

Comment:
These aren't types. Let's move these two up to come after Toast. We should export `useToast` hook as well. We also have to export ToastContext and ToastContextProvider.